docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
   postgres:
     image: postgres:12-alpine
     volumes:
-      - ./postgres-data:/var/lib/postgresql/data
+      - ./postgres-data:/var/lib/postgresql/data:z
```

Comment:
Whats this :z here?

Reply:
It doesn’t relate to Poetry, it shouldn’t be in this MR. 
It’s useful for Linux with SELinux so that the volume don’t have to be root in the local dev machine. https://stackoverflow.com/a/35222815/5091196